pub_sub/python-http/checkout/app.py

```diff
@@ -0,0 +1,27 @@
+import json
+import time
+import random
+import logging
+import requests
+import os
+
+logging.basicConfig(level=logging.INFO)
+
+base_url = os.getenv('BASE_URL', 'http://localhost') + ':' + os.getenv(
+                    'DAPR_HTTP_PORT', '3500')
+PUBSUB_NAME = 'order_pub_sub'
+TOPIC = 'orders'
+logging.info('Publishing to baseURL: %s, Pubsub Name: %s, Topic: %s' % (
+            base_url, PUBSUB_NAME, TOPIC))
+
+while True:
+    order = {'orderid': random.randint(1, 1000)}
```

Comment:
I assume we are keeping all small letters for python and keeping camel case for other languages?

Reply:
That's a bug good catch.  Fixing.

Reply:
Fixed